content/post/2019-03-26-why-care.md

```diff
@@ -0,0 +1,419 @@
+---
+date: "2019-03-26"
+title: R package developers, why should you care about R-hub?
+---
+
+**tl;dr: You should care about R-hub if you care about `R CMD check`, on all operating systems, for free, and don't ever want to leave R.**
+
+So, why use the R-hub package builder?
+
+## It is useful
+
+As an R package developer, you probably regularly use [`R CMD check`](http://r-pkgs.had.co.nz/check.html) to detect common problems in your package, and you might even [_love that command_](https://juliasilge.com/blog/how-i-stopped/). You might run it locally, or on a [continuous integration platform](https://juliasilge.com/blog/beginners-guide-to-travis/) such as Travis (Linux, macOS) or Appveyor (Windows). That's great! 
+
+Now, how can you add a platform to your arsenal once in a while, or make sure your package will behave correctly on CRAN's platforms? No need to borrow someone's machine or to tweak your machine or a continuous integration config file: the R-hub package builder is there for you! `R CMD check` as a service on all operating systems, with platforms corresponding to different settings. 
+
+## You do not need to leave R
+
+There is an R client, `rhub`, allowing to manage your package builds on R-hub platforms from your R console! By default, you can follow along how your package build is going:
+
+{{< figure src="/2019-03-18-why-care/check-output.gif" alt="check log in the R console" >}}
+
+
+Here's how you'd prepare a CRAN submission by submitting your package to all 3 recommended platforms, before getting a summary of results ready to be copy-pasted into your `cran-comments.md` file.
+
+```r
+chk <- check_for_cran()
+# wait a bit
+chk$cran_summary()
+group_check$cran_summary()
```

Comment:
Where is `group_check` coming from? I guess what you need here is
- look at the status of all three builds, maybe just by printing `chk`,
- once they are done, do `chk$cran_summary()`.

Reply:
oops yeah typo